entity-framework/core/what-is-new/ef-core-8.0/whatsnew.md

```diff
@@ -769,358 +769,13 @@ Azure SQL and SQL Server have a special data type called [`hierarchyid`](/sql/t-
 
 The database is then able to run queries against this data using its hierarchical structure. For example, a query can find ancestors and dependents of given items, or find all items at a certain depth in the hierarchy.
 
-#### HierarchyId support in .NET and EF Core
```

Comment:
Would very much prefer we didn't remove the info here when adding other docs. I would like this to remain a detailed record of what we shipped in EF7 and how things worked then.